tripy/nvtripy/frontend/trace/ops/pooling.py

```diff
@@ -200,7 +200,7 @@ def maxpool(
 @export.public_api(document_under="operations/functions")
 @wrappers.interface(
     dtype_constraints={"input": "T1", wrappers.RETURN_VALUE: "T1"},
-    dtype_variables={"T1": ["float32", "float16", "int8"]},
+    dtype_variables={"T1": ["float32", "float16"]},  # TODO: Add int8 once MLIR-TRT supports (issue #457)
```

Comment:
We have an unspoken convention for TODO's that mention issues (just makes it easier to grep):
```suggestion
    # TODO (#457): Add int8 once MLIR-TRT supports
    dtype_variables={"T1": ["float32", "float16"]},  
```